helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml

```diff
@@ -1,33 +0,0 @@
-{{- if .Values.syncservice.enabled }}
-{{- if .Values.syncservice.horizontalPodAutoscaling.enabled }}
```

Comment:
These values need to be removed from the `values.yaml` file

Reply:
Removed {{- if .Values.syncservice.enabled }} but syncservice.horizontalPodAutoscaling.enabled is not inside values file.